src/pages/Bets/components/BetRows.tsx

```diff
@@ -1,13 +1,18 @@
 import React from "react";
+import { useAccount } from "wagmi";
 import { BetHistory } from "../../../types";
 import { Row } from "./BetTable";
 
 type Props = {
+  myBetsSelected: boolean;
   bets?: BetHistory[];
   onClickBet: (bet?: BetHistory) => void;
 };
 
-const BetRows: React.FC<Props> = ({ bets, onClickBet }) => {
+const BetRows: React.FC<Props> = ({ myBetsSelected, bets, onClickBet }) => {
+  const { isConnected } = useAccount();
+  if (!isConnected && myBetsSelected)
```

Comment:
```suggestion
  if (!isConnected)
```

Reply:
the user can be not connect and still want to see total bets, we only want to show that they arent connected if they are trying to view "My Bets"